bindings/py/nupic/bindings/__init__.py

```diff
@@ -22,4 +22,4 @@
 """
 ## @file
 """
-__import__('pkg_resources').declare_namespace(__name__)
+# __import__('pkg_resources').declare_namespace(__name__)
```

Comment:
You want to really delete the line. We don't want to leave commented-out code bits in the repo. This confuses everyone down the road.